app/assets/stylesheets/components/_notification.sass

```diff
@@ -0,0 +1,12 @@
+.Notification
+  width: 33%
+  padding: 10px 0
```

Comment:
*10px*
That's a Magic Number. By the way regarding code smells: [read as well](http://csswizardry.com/2012/11/code-smells-in-css/)